modules/nw-cudn-best-practices.adoc

```diff
@@ -28,4 +28,12 @@ Before setting up a `ClusterUserDefinedNetwork` custom resource (CR), users shou
 
 ** If the namespace is missing the `k8s.ovn.org/primary-user-defined-network` label and a primary `ClusterUserDefinedNetwork` CR already exists, a pod in the namespace is created and attached to the default network.
 
-** If the namespace _has_ the label, and a primary `ClusterUserDefinedNetwork` CR does not exist, a pod in the namespace is not created until the `ClusterUserDefinedNetwork` CR is created.
\ No newline at end of file
+** If the namespace _has_ the label, and a primary `ClusterUserDefinedNetwork` CR does not exist, a pod in the namespace is not created until the `ClusterUserDefinedNetwork` CR is created.
+
+* When using the `ClusterUserDefinedNetwork` CR to create `localnet` topology, the following are best practices for administrators:
+
+** You must make sure that the `spec.network.physicalNetworkName` field matches the field that you configured in the OVS bridge mapping when you create your CUDN. This ensures that you are bridging to the intended segment of your physical network. Do not reuse the same `physicalNetworkName` for different physical networks.
+
+** Avoid overlapping subnets between your physical network and your other network interfaces. Overlapping network subnets can cause routing conflicts and network instability.
```

Comment:
I think it should mention that when OVN-K IPAM is being used (i.e.: when `spec.nework.localnet.subsets` is specified) one could prevent such conflicts by specifying the reserved IPs/subnets in the `spec.nework.localnet.excludeSubnets` attribute.

Reply:
I rewrote to try to clarify.

Reply:
@maiqueb here is comment where I introduced the mistake came from. I'll fix, sorry.